design-system/materials/internals/definition.yaml

```diff
@@ -170,9 +174,7 @@ choiceGroupsByTheme:
       prefix: color
       description: Dark theme colors overwrite
       choices:
-        color-primary: '#f78d26'
-        color-solid: '#fff'
-        color-surface: '#1a1a1a'
+        color-primary: '#00b39e'
```

Comment:
Currently the generate tokens script needs the test section to exist so I'm just copying one of the values here.
Maybe we could look into it as a follow-up.